src/systemcmds/tune_control/tune_control.cpp

```diff
@@ -62,18 +64,40 @@ extern "C" {
 static void
 usage()
 {
-	PX4_INFO(
-		"External tune control for testing. Usage:\n"
-		"tune_control play [-t <default tunes>] [-f <frequency>] [-d <duration>] [-s <strength>] [-m < melody>]\n"
-		"\n"
-		"\t-t <defualt tunes>\tPlay the default (1...15) (default=1)\n"
-		"\t-f <frequency>\t\tFrequency from 0-20kHz\n"
-		"\t-d <duration>\t\tDuration of the tone in us\n"
-		"\t-s <strength>\t\tStrength of the tone between 0-100\n"
-		"\t-m <melody>\t\tMelody in a string form ex: \"MFT200e8a8a\"\n"
-		"\n"
-		"tune_control stop \t\tStops playback, useful for repeated tunes\n"
-	);
+
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
+### Description
+
+Command-line tool to control & test the (external) tunes.
+
+Tunes are used to provide audible notification and warnings (e.g. when the system arms, gets position lock, etc.).
+The tool requires that a driver is running that can handle the tune_control uorb topic.
+
+Information about the tune format and predefined system tunes can be found here:
+https://github.com/PX4/Firmware/blob/master/src/lib/tunes/tune_definition.desc
+
+### Examples
+
+Play system tune #2:
+
+    tune_control play -t 2
+
```

Comment:
might be a bit much white space here?

Reply:
I typically do:
Play system tune #2:
$ tune_control play -t 2
Which then will be converted to use the backticks for markdown output.

Reply:
Thanks. Done! Merge if you're happy.